scipy/sparse/bsr.py

```diff
@@ -407,14 +407,14 @@ def tobsr(self,blocksize=None,copy=False):
         else:
             return self
 
-    def tocsr(self):
-        return self.tocoo(copy=False).tocsr()
+    def tocsr(self, copy=False):
+        return self.tocoo(copy=copy).tocsr()
         # TODO make this more efficient
 
-    def tocsc(self):
-        return self.tocoo(copy=False).tocsc()
+    def tocsc(self, copy=False):
+        return self.tocoo(copy=copy).tocsc()
```

Comment:
As above, I would prefer to use explicit values for `copy` parameter in this case as well.

And thanks for the comments. :)